ramls/item.json

```diff
@@ -128,6 +132,34 @@
             "type": "string",
             "description": "Text to display"
           },
+          "source": {
+            "type": "object",
+            "description": "The user who added the note",
+            "properties": {
```

Comment:
As these properties are stored copies of information from the user (presumably to support sorting), they should be marked read only, as this behaviour is controlled by the server.